.dockerignore

```diff
@@ -0,0 +1,2 @@
+bin/
+obj/
```

Comment:
Should this have a newline before EOF?

Reply:
Didn't seem to break anything when I locally built the docker image, however that was done on my Windows machine. Not sure if it would break anything on linux.